src/os/ui/filter/op/istrueop.js

```diff
@@ -0,0 +1,62 @@
+goog.provide('os.ui.filter.op.IsTrue');
+
+goog.require('os.ui.filter.op.Op');
+
+
+/**
+ * A 'PropertyIsTrue' operation class.
+ * Based on the OGC Filter Spec
+ *
+ * @extends {os.ui.filter.op.Op}
+ * @constructor
+ */
+os.ui.filter.op.IsTrue = function() {
+  os.ui.filter.op.IsTrue.base(this, 'constructor',
+      'And', 'is true', 'true', undefined, undefined, 'Supports true, >1, and "true"', 'span', true);
```

Comment:
`>=1` ?

Reply:
Ah, yup that comment is doubly wrong.  I changed it to ==1 to make the implementation less flexible; since this isn't implemented like a coding "boolean" (where non-nulls/non-empty's are true unless explicitly false, etc).